source/howto/remote-access/pycharm.rst

```diff
@@ -0,0 +1,141 @@
+
+.. meta::
+    :description: How to attach a remote debugger from PyCharm to a Valohai execution
+
+.. _remote-ssh-pycharm:
+
+Debug from PyCharm
+##################
+
+.. warning::
+
+    This guide assumes you're already familiar with the fundamentals of Valohai and are working with a project that's connected to a Git-repository, and configured on Valohai.
```

Comment:
Nitpick: I don't think git repository is spelled together like that. 🤔

Reply:
It's not. 😅

Reply:
(Also the ones below)